.github/workflows/e2e_scripts/cleanup_monitoring.py

```diff
@@ -0,0 +1,121 @@
+    """ Deletes all dashboards. """
+    client = v1.DashboardsServiceClient()
+    dashboards = list(client.list_dashboards(project_name))
+    while (len(dashboards) != 0):
+        for dashboard in dashboards:
+            try:
+                client.delete_dashboard(dashboard.name)
+            except:
+                print('Could not delete dashboard: ' + dashboard.name)
+        dashboards = list(client.list_dashboards(project_name))
+
+def cleanupPolicies():
+    """ Delete all alerting policies for both uptime checks and SLOs. """
+    client = monitoring_v3.AlertPolicyServiceClient()
+    policies = list(client.list_alert_policies(project_name))
+    while (len(policies) != 0):
+        for policy in policies:
+            try:
+                client.delete_alert_policy(policy.name)
+            except:
+                print('Could not delete alerting policy: ' + policy.name)
+        policies = list(client.list_alert_policies(project_name))
+
+def cleanupNotificationChannels():
+    """ Deletes all notification channels. """
+    client = monitoring_v3.NotificationChannelServiceClient()
+    channels = list(client.list_notification_channels(project_name))
+    while (len(channels) != 0):
+        for channel in channels:
+            try:
+                client.delete_notification_channel(channel.name)
+            except:
+                print('Could not delete notification channel: ' + channel.name)
+        channels = list(client.list_notification_channels(project_name))
+
+def cleanupServices():
+    """ Deletes only custom services that are identified by a trailing 'srv' in the name. """
+    client = monitoring_v3.ServiceMonitoringServiceClient()
+    # only delete custom services
+    custom_services = [s for s in client.list_services(project_name) if s.name.endswith("srv")]
+    while (len(custom_services) != 0):
+        for service in custom_services:
+            try:
+                client.delete_service(service.name)
+            except:
+                print('Could not delete service: ' + service.name)
+        custom_services = [s for s in client.list_services(project_name) if s.name.endswith("srv")]
+
+def cleanupSlos():
+    """ Deletes every SLO associated with every service. """
+    client = monitoring_v3.ServiceMonitoringServiceClient()
+    slos = [slo for service in client.list_services(project_name) for slo in client.list_service_level_objectives(service.name)]
+    while (len(slos) != 0):
+        for slo in slos:
+            try:
+                client.delete_service_level_objective(slo.name)
+            except:
+                print('Could not delete SLO: ' + slo.name)
+        slos = [slo for service in client.list_services(project_name) for slo in client.list_service_level_objectives(service.name)]
+
+def cleanupUptimeCheck():
+    """ Deletes every uptime check. """
+    client = monitoring_v3.UptimeCheckServiceClient()
+    uptime_checks = list(client.list_uptime_check_configs(project_name))
```

Comment:
nit: the better Python practice would probably be to name `project_name` an argument for each function instead of a global variable. But I don't think that's too important for our purposes

Reply:
It was a small change so I made it!